nilearn/surface/surface.py

```diff
@@ -1869,20 +1886,17 @@ class SurfaceImage:
            :obj:`pathlib.Path`
            Data for the both hemispheres.
 
-    squeeze_on_save : :obj:`bool` or None, default=None
-            If ``True`` axes of length one from the data
-            will be removed before saving them to file.
-            If ``None`` is passed,
-            then the value will be set to ``True``
-            if any of the data parts is one dimensional.
+    dtype : DTypeLike object, default=None
+        dtype to enforce on the data.
```

Comment:
At the moment, this PR does not check if the mesh have consistent dtype across hemispheres but I feel this is less important.